langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -113,12 +172,22 @@ public isolated function abs(float x) returns float = @java:Method {
 # (this is the round-to-nearest rounding mode, which is the default for IEEE and for Ballerina).
 # A value of `fractionDigits` greater than 0 thus corresponds to the number of digits after the decimal
 # point being `fractionDigits`; a value of 0 for `fractionDigits` rounds to an integer.
-# If `x` is NaN, +0, -0, +∞ or -∞, then the result is `x``.
+# If `x` is NaN, +0, -0, +∞ or -∞, then the result is `x`.
 # When `fractionDigits` is 0, this is
 # the same as Java Math.rint method, .NET Math.Round method and
 # IEEE roundToIntegralTiesToEven operation
 # Note that `<int>x` is the same as `<int>x.round(0)`.
 #
+# ```ballerina
+# float score = 3.55;
+# // Round a float number to the nearest integer and return as a float.
```

Comment:
I think we can just say

```suggestion
# // Round a float number to zero fractional digits.
```

Since the description explains it a bit.

Reply:
Updated.